spec/features/admin/manage_organization_spec.rb

```diff
@@ -539,7 +539,10 @@
         let!(:cross_sell_to) { create(:market) }
 
         before do
-          seller.update_cross_sells!(from_market: market, to_ids: [cross_sell_to.id])
+          UpdateCrossSellingMarketOrganizations.perform(
```

Comment:
@thermistor is there any difference in using `call` vs `perform` for an interactor?

Reply:
Ah nevermind, it was `CoerceSourceToDestinationMaps` but looking now I see that isn't actually an interactor class.